kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/client/ServiceDiscovery.scala

```diff
@@ -258,6 +255,7 @@ case class ServiceNodeInfo(
     nodeName: String,
     host: String,
     port: Int,
-    version: Option[String]) {
+    version: Option[String],
+    sessionId: Option[String]) {
```

Comment:
how about naming this and zk node `createSessionId` FYI @yaooqinn

Reply:
make sense.